2차/윤형진_가운데-글자-가져오기.py

```diff
@@ -0,0 +1,10 @@
+def solution(s):
+    answer = ''
+    word_len = len(s)
+    
+    if word_len % 2 == 0 :
+        answer = s[(word_len // 2 - 1) :(word_len // 2 + 1)]
+    else :
+        answer = s[word_len // 2]
+    
+    return answer
```

Comment:
구조를 분해하는게 좋을 것 같아요
생각나는 방법은 두가지가 있네요

```suggestion
    if word_len % 2 == 0 :
        return s[(word_len // 2 - 1) :(word_len // 2 + 1)]
    return s[word_len // 2]
```

```suggestion
    return s[(word_len // 2 - 1) :(word_len // 2 + 1)] if word_len % 2 == 0 else s[word_len // 2]
```